mozdownload/__init__.py

```diff
@@ -2,4 +2,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from scraper import *
+from scraper import * #noqa
```

Comment:
Given that wild-card imports are considered bad, you can import each scraper class individually and add them to a `__all__` array.